server/src/controllers/mapPreview/index.ts

```diff
@@ -28,15 +28,13 @@ const getMapPreview = async (ctx: Context) => {
     }
 
     const newResponse = new Response(response.body, {
+      status: 200,
       headers: {
         'Content-Type': 'image/png',
       },
     });
 
-    console.log('newResponse', newResponse);
-
-    // return newResponse;
-    return ctx.json(newResponse, 200);
```

Comment:
@MuhammadHassan03 @andrew-bierman  I deleted this line because the response is not a json and should not be converted to json value. we expect an image.